src/middlewared/middlewared/common/ports/__init__.py

```diff
@@ -19,6 +19,7 @@ async def get_ports(self):
 
 
 class ServicePortDelegate(PortDelegate):
+    bind_address_field = NotImplementedError
```

Comment:
`NotImplementedError` is an exception class. There is a `NotImplemented` constant just for this purpose (or why not just `None`?)